cmake/modules/FindPETSC.cmake

```diff
@@ -197,16 +197,28 @@ IF(NOT PETSC_PETSCVARIABLES MATCHES "-NOTFOUND")
   ENDFOREACH()
 ENDIF()
 
+IF(PETSC_WITH_MPIUNI)
+  #
+  # Workaround: Some loser distributions do not install petscvariables and
```

Comment:
While I don't agree with what Fedora does either, I think we should not describe these distributions so harshly. One doesn't know who will ultimately read these things.